lib/libexif/ExifParse.cpp

```diff
@@ -480,6 +481,15 @@ void CExifParse::ProcessDir(const unsigned char* const DirStart,
       }
       break;
 
+      case TAG_XP_COMMENT:
+      {
+        // The XP user comment is allows unicode encoded
```

Comment:
This sentence sounds strange to me. Did you perhaps mean "always" instead of "allows"?